spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -36,35 +35,35 @@ def assert_common
 
   def expected_table_counts
     {
-      :auth_private_key              => 12,
```

Comment:
I'm starting to think if those numbers make sense at all.
Did you check if they are correct? So they dont provide any value.

Can we create them by running an api call?
For `:vm` it should be just `client.describe_instances.count`